### PR TITLE
Dump sample - accession mappings, and update for r1-15

### PR DESCRIPTION
**Pull request by wasade**
@ElDeveloper @adamrp, possible for quick review? This method isn't in place and the r1-15 sample mappings aren't in the repo

### CODE REVIEW

File: scripts/ag [hunk 1]
```diff
@@ -54,5 +54,30 @@ def persample_json(table, output):
             fp.write(json.dumps(tree, indent=2))
 
 
+@ag.command()
+@click.argument('accession-metadata', required=True, nargs=-1)
+def sample_to_accession(accession_metadata):
+    """Dump {accession: [sample]} and {sample: accession}"""
```
Comment (ElDeveloper) [inline scripts/ag]: Why is it a list of a single sample? Why not just the sample itself?

File: scripts/ag [hunk 2]
```diff
@@ -54,5 +54,30 @@ def persample_json(table, output):
...
+    accession_to_sample = {}
+    sample_to_accession = {}
+
+    for f in accession_metadata:
+        accession = os.path.basename(f).rsplit('.', 1)[0]
+        accession_to_sample[accession] = []
+        with open(f) as fp:
```
Comment (ElDeveloper) [inline scripts/ag]: Worth opening in 'U' mode?

**Comment by ElDeveloper** [comment]
:scream: Travis says all checks failed, but the code looks good at a first glance. I don't have a dev environment to test this in though.

File: scripts/ag (hunk 1 above)
Comment (wasade) [inline scripts/ag]: It's `{accession: [list, of, samples]}`

File: scripts/ag (hunk 2 above)
Comment (wasade) [inline scripts/ag]: eh, we control their production from within the code base

File: scripts/ag (hunk 1 above)
Comment (ElDeveloper) [inline scripts/ag]: Ahhh, would you mind clarifying that doctor? :briefcase:

File: scripts/ag (hunk 2 above)
Comment (ElDeveloper) [inline scripts/ag]: Got it.

File: scripts/ag (hunk 1 above)
Comment (wasade) [inline scripts/ag]: Isn't that one of the normal notations to see? Like `[int]` implies `list` of `int`. Can change if you'd like tho

Comment (ElDeveloper) [inline scripts/ag]: Hmmm, I guess I wasn't familiar with that notation, if it's standard
then I should revise my ways, so yeah just go with whatever is standard
(which seems like the current approach).

**Comment by wasade** [comment]
Passing now, so up to you!! :)

**Pull request merged**